frontend/class-public.php

```diff
@@ -274,10 +274,7 @@ public function display_volunteer_opp_list( $list_type, $query_args ){
 
 			<?php } ?>
```

Comment:
You don't have to close the PHP tag here since the line is still using PHP now. I understand if you want to keep it this way if you think it looks cleaner.